salt/_grains/etcd_info.py

```diff
@@ -0,0 +1,40 @@
+try:
+    import etcd
+    HAS_LIBS = True
+except ImportError:
+    HAS_LIBS = False
+
+from salt.exceptions import CommandExecutionError
+
+def etcd_info():
+
+  etcd_info = {}
+  etcd_info['members_all'] = []
+  etcd_info['member_type'] = "proxy"
+
+  if HAS_LIBS:
+    etcd_info['available'] = True
+    client = etcd.Client(host='127.0.0.1', port=2379)
```

Comment:
Hmmm, I'm not sure about TLS here, right now we are supporting only etcd clientURLs with HTTP.